### PR TITLE
Added links to repository and issues section

### PR DESCRIPTION
**Pull request by BhaswatiRoy**
Fix: https://github.com/lfortran/lcompilers_frontend/issues/6
Added links in the footer to direct to the main repository and issues section for contribution and raising issues.

### CODE REVIEW

**Comment by Shaikh-Ubaid** [comment]
Currently, the footer looks as follows on my local run:
image



Shall we use a `pipe` character to separate the links?:
```bash
diff --git a/components/MyFooter.js b/components/MyFooter.js
index e762482..d183267 100644
--- a/components/MyFooter.js
+++ b/components/MyFooter.js
@@ -12,7 +12,10 @@ function MyFooter() {
                 width: "100%",
             }}
         >
-            <a href="https://lfortran.org/">LFortran</a> (<a href={"https://github.com/lfortran/lfortran/commit/" + commit.id}>{commit.id}</a>) by <a href="https://lcompilers.org/">LCompilers</a>
+            <a href="https://lfortran.org/">LFortran</a> (<a href={"https://github.com/lfortran/lfortran/commit/" + commit.id}>{commit.id}</a>) by &nbsp;
+            <a href="https://lcompilers.org/">LCompilers</a> | &nbsp;
+            <a href="https://github.com/lfortran/lcompilers_frontend">Contribute</a> |&nbsp;
+            <a href="https://github.com/lfortran/lcompilers_frontend/issues">Report Issues</a>
         </Footer>
     );
 }
```
The output looks as follows (On desktop): 
image

(On Phones):
image

**Comment by Shaikh-Ubaid** [comment]
For links to `Contribute` and `Report Issues`, shall we have links to https://github.com/lfortran/lfortran or https://github.com/lfortran/lcompilers_frontend?

**Review by Shaikh-Ubaid — APPROVED** [review]
It looks good to me. Thank you so much Bhawasti Roy for this.

File: components/MyFooter.js [hunk 1]
```diff
@@ -12,7 +12,10 @@ function MyFooter() {
                 width: "100%",
             }}
         >
-            <a href="https://lfortran.org/">LFortran</a> (<a href={"https://github.com/lfortran/lfortran/commit/" + commit.id}>{commit.id}</a>) by <a href="https://lcompilers.org/">LCompilers</a>
+            <a href="https://lfortran.org/">LFortran</a> (<a href={"https://github.com/lfortran/lfortran/commit/" + commit.id}>{commit.id}</a>) by &nbsp;
++            <a href="https://lcompilers.org/">LCompilers</a> | &nbsp;
++            <a href="https://github.com/lfortran/lcompilers_frontend">Contribute</a> |&nbsp;
++            <a href="https://github.com/lfortran/lcompilers_frontend/issues">Report Issues</a>
```
Comment (Shaikh-Ubaid) [inline components/MyFooter.js:18]:
```suggestion
            <a href="https://lfortran.org/">LFortran</a> (<a href={"https://github.com/lfortran/lfortran/commit/" + commit.id}>{commit.id}</a>) by &nbsp;
            <a href="https://lcompilers.org/">LCompilers</a> | &nbsp;
            <a href="https://github.com/lfortran/lcompilers_frontend">Contribute</a> |&nbsp;
            <a href="https://github.com/lfortran/lcompilers_frontend/issues">Report Issues</a>
```
Sorry, there is a minor change needed. There are some unneeded `+` symbols in the code here. I noticed them just now.

**Comment by BhaswatiRoy** [comment]
Hello @Shaikh-Ubaid 
Thanks for pointing that out. I have removed them

**Review by Shaikh-Ubaid — APPROVED** [review]
It looks great! Thank you so much Bhawasti Roy!

**Review by certik — APPROVED** [review]

**Pull request merged**